### PR TITLE
bgpv2: Status reporting document

### PR DESCRIPTION
**Pull request by YutaroHayakawa**
Document recently added status conditions and `bgpControlPlane.statusReport.enabled` flag.

```release-note
bgpv2: Status reporting document
```

### CODE REVIEW

File: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst [hunk 1]
```diff
@@ -175,6 +203,15 @@ In the following example, you can see BGP instance state from node ``bgpv2-cplan
             Applied Keepalive Seconds:  30
     Events:                             <none>
 
+Disabling CRD Status Report
+===========================
+
+CRD status reporting is useful for troubleshooting. Thus, it is recommended to
+enable it in general. However for the large cluster which has a lot of nodes or
```
Comment (rastislavs) [inline Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst]:
```suggestion
enable it in general. However, for large clusters which have a lot of nodes or
```

**Review by rastislavs — APPROVED** [review]
LGTM with small nit, thanks!

File: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst (hunk 1 above)
Comment (YutaroHayakawa) [reply Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst]: Thanks, fixed!

File: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst [hunk 2]
```diff
@@ -175,6 +203,15 @@ In the following example, you can see BGP instance state from node ``bgpv2-cplan
...
 
+Disabling CRD Status Report
+===========================
+
+CRD status reporting is useful for troubleshooting. Thus, it is recommended to
+enable it in general. However, for large clusters which have a lot of nodes or
+BGP policies, it may make a significant API server load. If you wish to disable
+status reporting, you can set ``bgpControlPlane.statusReport.enabled`` Helm
+value to ``false``. It disables the status reporting and clears the currently
+reported status.
```
Comment (zacharysarah) [inline Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst]: Tweaks for active voice and clarity
```suggestion
CRD status reporting is useful for troubleshooting, making it useful to
enable in general. However, for large clusters with a lot of nodes or
BGP policies, CRD status reporting may add a significant API server load. 
To disable status reporting, set the ``bgpControlPlane.statusReport.enabled`` Helm
value to ``false``. Doing so disables status reporting and clears the currently
reported status.
```

**Review by zacharysarah — COMMENTED** [review]
@YutaroHayakawa Nice work. ⭐ The one set of changes I recommend is nice to have but not blocking.

File: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst (hunk 2 above)
Comment (YutaroHayakawa) [reply Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst]: Applied, thanks!

**Pull request merged**